Simplenote/Verification/AccountVerificationRemote.swift

```diff
@@ -0,0 +1,55 @@
+import Foundation
+
+// MARK: - AccountVerificationRemote
+//
+class AccountVerificationRemote {
+    private let urlSession: URLSession
+
+    init(urlSession: URLSession = URLSession.shared) {
+        self.urlSession = urlSession
+    }
+
+    /// Send verification request for specified email address
+    ///
+    func verify(email: String, completion: @escaping (_ success: Bool) -> Void) {
+        guard let request = verificationURLRequest(with: email) else {
+            completion(false)
+            return
+        }
+
+        let dataTask = urlSession.dataTask(with: request) { (data, response, error) in
+            DispatchQueue.main.async {
+                // Check for 2xx status code
+                guard let response = response as? HTTPURLResponse, response.statusCode / 100 == 2 else {
+                    completion(false)
+                    return
```

Comment:
just a note that I've designed the API to _always_ send a 200 regardless of what happens. that means the only time we shouldn't get a 200 is when an issue on the server sends a 5xx.

in those cases it might be hard to figure out what's wrong or do anything with the knowledge.

could be easier to "fire and forget"

did you have a response based on the status code you wanted to do?

Reply:
Oh I see. I believe when I first checked the code some time ago I saw that that endpoint could return 4xx. 

When status code is not 2xx or there is no status code at all in case of network errors we show a generic error alert.

<img width="375" alt="Screenshot 2021-01-21 at 09 16 17" src="https://user-images.githubusercontent.com/54751/105322818-a9304880-5bc9-11eb-8c4f-3bf4a38fa799.png">